command/root.go

```diff
@@ -363,25 +367,72 @@ func determineEditor(cmd *cobra.Command) (string, error) {
 	return editorCommand, nil
 }
 
-func ExpandAlias(args []string) ([]string, error) {
-	empty := []string{}
+func ExecuteShellAlias(args []string) error {
+	externalCmd := exec.Command(args[0], args[1:]...)
+	externalCmd.Stderr = os.Stderr
+	externalCmd.Stdout = os.Stdout
+	externalCmd.Stdin = os.Stdin
+	preparedCmd := run.PrepareCmd(externalCmd)
+
+	err := preparedCmd.Run()
+	if err != nil {
+		return fmt.Errorf("failed to run external command: %w", err)
```

Comment:
When a shell alias failed, right now lots of extra output is printed to stderr. Here is an alias that expands to `false`:

```
> bin/gh g
failed to run alias ["C:\\Program Files\\Git\\bin\\sh.exe" "-c" "false"]: failed to run external command: exit status 1
```

I'm thinking that we could trust that the commands used in the alias will print their own error messages to stderr and that we don't need to print anything additionally to stderr. Basically, if an alias failed, we could just ensure to exit the main process with the same exit code as the alias and print nothing extra to stderr. How do you feel about that?

Reply:
I think that's totally reasonable; especially since if `DEBUG` is on the user would see the alias expansion anyway.